src/App.js

```diff
@@ -42,20 +102,77 @@ function App() {
       );
     }
   }, [currentCoordinates]);
-  useEffect(() => {
-    LocationService.getCoordinatesForSearch("polonia").then((places) => {
-      setPlaces(places);
-    });
-  }, []);
+  // useEffect(() => {
+  //   LocationService.getCoordinatesForSearch("polonia").then((places) => {
+  //     setPlaces(places);
+  //   });
+  // }, []);
 
   return (
     <div className="App">
-      <h1>
-        {city}/{country}
-      </h1>
-      <h2>Temperatura Hoy: {temp}°</h2>
-      <h2>Temperatura Mañana: {dailyTemp}°</h2>
-      <h3>Prueba:{places[0].cityName}</h3>
+      <div className="actualCity">
+        <LocationLabel city={city} country={country} />
+      </div>
+      <div className="dayActual">
+        <Day
+          //temp={temp}
+          //ruana="ruana"
+          icon={dayCurrentIcon}
+          weekDay={weekDays[weekDay]}
+          day={day}
+          month={months[month]}
+        />
+      </div>
+      <div className="firstDay">
+        <Day
+          //temp={dailyTemp}
+          //ruana="ruana"
+          icon={dailyIcon[1]}
+          weekDay={weekDays[dailyWeekDay[1]]}
+          day={dailyDay[1]}
+          month={months[dailyMonth[1]]}
+        />
+      </div>
+      <div className="secondDay">
+        <Day
+          //temp={dailyTemp}
+          //ruana="ruana"
+          icon={dailyIcon[2]}
+          weekDay={weekDays[dailyWeekDay[2]]}
+          day={dailyDay[2]}
+          month={months[dailyMonth[2]]}
+        />
+      </div>
+      <div className="thirdDay">
+        <Day
+          //temp={dailyTemp}
+          //ruana="ruana"
+          icon={dailyIcon[3]}
+          weekDay={weekDays[dailyWeekDay[3]]}
+          day={dailyDay[3]}
+          month={months[dailyMonth[3]]}
+        />
+      </div>
+      <div className="fourthDay">
+        <Day
+          //temp={dailyTemp}
+          //ruana="ruana"
+          icon={dailyIcon[4]}
+          weekDay={weekDays[dailyWeekDay[4]]}
+          day={dailyDay[4]}
+          month={months[dailyMonth[4]]}
+        />
+      </div>
+      <div className="fifthDay">
+        <Day
+          //temp={dailyTemp}
+          //ruana="ruana"
+          icon={dailyIcon[5]}
+          weekDay={weekDays[dailyWeekDay[5]]}
+          day={dailyDay[5]}
+          month={months[dailyMonth[5]]}
+        />
+      </div>
```

Comment:
great, I only would recommend using .map() to avoid repeating code unnecessarily.